src/components/PostsList.tsx

```diff
@@ -1,54 +1,56 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-
-import classNames from 'classnames';
 import React from 'react';
 import { Post } from '../types/Post';
+import classNames from 'classnames';
```

Comment:
Ensure that the `Post` type is correctly defined and imported from the `../types/Post` file. Verify its existence and correctness.